skforecast/ForecasterRnn/utils.py

```diff
@@ -0,0 +1,127 @@
+################################################################################
+#                             skforecast.utils                                 #
+#                                                                              #
+# This work by skforecast team is licensed under the BSD 3-Clause License      #
+################################################################################
+# coding=utf-8
+
+from typing import Union, Any, Optional, Tuple, Callable
+import pandas as pd
+
+import tensorflow as tf
+from tensorflow import keras
+from keras.models import Model
+from keras.layers import Dense, Input, Reshape, LSTM, SimpleRNN
+from keras.optimizers import Adam
+from keras.losses import MeanSquaredError
+
+
+def create_and_compile_model(
+    series: pd.DataFrame,
+    lags: Union[int, list],
+    steps: Union[int, list],
+    levels: Union[str, int, list] = None,
+    recurrent_layer: str = "LSTM",
+    recurrent_units: Union[int, list] = 100,
+    dense_units: list = [64],
+    activation: str = "relu",
+    optimizer: object = Adam(learning_rate=0.01),
+    loss: object = MeanSquaredError(), 
+    # compile_kwars: dict = {} # TODO. menor prioridad
```

Comment:
@JoaquinAmatRodrigo , @JavierEscobarOrtiz 

Who should take precedence, compile_kwars or the parameters inside the function?

```
optimizer: object = Adam(learning_rate=0.01),
loss: object = MeanSquaredError(),
```
If I give more priority to the parameters inside the function, the default values (e.g., `loss: object = MeanSquaredError`) will override those passed in compile_kwars, right?

Reply:
@FernandoCarazoMelo 

Please, change `compile_kwars` to  `compile_kwargs`

Yes, give priority to the parameters inside the function. If at any time `optimizer` or `loss` are inside `compile_kwargs`, they should be overwritten with the parameter values.

Reply:
We can add a warning if the argument  optimizer and loss are defined  inside compile_kwargs. Indicating that it is overwritten with the argument of the function.